flink-kubernetes/pom.xml

```diff
@@ -35,17 +35,6 @@ under the License.
 		<kubernetes.client.version>4.9.2</kubernetes.client.version>
 	</properties>
 
-	<!-- Set dependency version for transitive dependencies -->
-	<dependencyManagement>
-		<dependencies>
-			<dependency>
-				<groupId>com.squareup.okhttp3</groupId>
-				<artifactId>okhttp</artifactId>
-				<version>3.12.1</version>
```

Comment:
I assume this upgrade to be safe because it only jumps by 2 minor versions, and I didn't see anything major in the release notes.